"use strict";

(function testError() {
    try {
        func();
    } catch (error) {
        return console.log("что-то пошло не так");
    } finally {
        return 0;
    }
})();


/* Задача 1
Создать блок обработки ошибок - при вызове несуществующей функции 
показывать пользователю на экране ошибку "что-то пошло не так" */